### PR TITLE
feat: add proxy api in config page debug page

### PR DESCRIPTION
**Pull request by laojun**
## What type of PR is this?

- [x] Feature
- [ ] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## What this PR does / why we need it:
feat: add proxy api in config page debug page

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [ ] No

## Special notes for your reviewer:
image

### CODE REVIEW

File: shell/app/config-page/components/table-group/table-group.tsx [hunk 1]
```diff
@@ -37,14 +37,7 @@ const TableBoard = (props: CP_TABLE_GROUP.ITableBoardProps) => {
...
           </div>
-          <Table
-            tableLayout="fixed"
-            props={table?.props}
-            data={table?.data}
-            operations={table?.operations}
-            {...extraProps}
-            type="Table"
-          />
+          <Table props={table?.props} data={table?.data} operations={table?.operations} {...extraProps} type="Table" />
```
Comment (McDaddy) [inline shell/app/config-page/components/table-group/table-group.tsx:40]: does antd table has this attribute? `type="Table"`

File: shell/app/config-page/mock/index.tsx [hunk 2]
```diff
@@ -14,20 +14,20 @@
...
-export const useMock =
-  (key: string, enhanceFunc: Function = noop) =>
-  (payload: Obj) => {
-    if (process.env.NODE_ENV === 'production') {
-      return Promise.resolve();
-    } else {
-      return import(`./${key}.mock`).then((file) => enhanceFunc(file.default, payload));
-    }
-  };
+const noop = (a: any) => a;
```
Comment (McDaddy) [inline shell/app/config-page/mock/index.tsx:23]: remove noop

File: shell/webpack.config.js [hunk 3]
```diff
@@ -252,6 +252,7 @@ module.exports = () => {
       ),
       new webpack.DefinePlugin({
         'process.env.NODE_ENV': JSON.stringify(nodeEnv),
+        'process.env.UI_ENV': JSON.stringify(process.env.erda_ui_env),
```
Comment (McDaddy) [inline shell/webpack.config.js:255]: where is the definition of `erda_ui_env`

File: shell/app/config-page/components/table-group/table-group.tsx (hunk 1 above)
Comment (laojun) [reply shell/app/config-page/components/table-group/table-group.tsx]: it's not a antd Table

File: shell/webpack.config.js (hunk 3 above)
Comment (laojun) [reply shell/webpack.config.js:255]: it's a runtime variable, defined in application pipeline setting

**Review by daskyrk — APPROVED** [review]

**Pull request merged**